### PR TITLE
Introducing ClusterLink blog

### LINKED ISSUE #495: ClusterLink introductory blog
- Discuss the motivation for clusterlink (e.g., compare with ingress access, resource efficiency, complexity, ...)
- Goal: Enable a simple way to manage secure connectivity between workloads in multiple clusters. 
Provide the management and security that is on par with the equivalent intra-cluster functionality (e.g., network policies, creation of service objects).
- List potential roadmap items to get feedback? (do we need a `ROADMAP.md` file in the repo?)

### PR DESCRIPTION
**Pull request by elevran**
Fixes #495

### CODE REVIEW

**Comment by elevran** [comment]
@michalmalka any initial thoughts?

File: website/content/en/blog/clusterlink-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ aspects must be addressed: orchestration (sometimes referred to as *"scheduling"*)
+ and connectivity. There are several existing open source projects providing these
+ capabilities. For example, projects such as [Open Cluster Management](https://open-cluster-management.io/)
+ and [KubeStellar](https://kubestellar.io) simplify orchestration and management of
+ workloads across multiple clusters.
+
+In order to facilitate cross cluster communications, you could use Kubernetes
+ native resources, including [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+ and [Gateway API](https://kubernetes.io/docs/concepts/services-networking/gateway/).
+ Or, alternatively, chose from a number of existing open source projects, such
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:29]:
```suggestion
 Or, alternatively, choose from a number of existing open source projects, such
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+ The goal is, to the extent possible, extend the Kubernetes single cluster network
+ abstraction to multicluster use cases.
+
+The creation of a shared mesh is not always desirable and may place additional
+ constraints on administrators, developers and the workloads they manage. For example,
+ it might make assumptions on objects, such as Services, being the "same" based
+ on the objects sharing a name (i.e., namespace sameness as defined
+ [here](https://istio.io/latest/docs/ops/configuration/traffic-management/multicluster/)
+ or [here](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)),
+ or require planning in assignment of IP addresses across independent clusters, etc.
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:42]:
```suggestion
 or require planning in assignment of IP addresses across independent clusters.
```
[No need for "etc." when using "For example"](https://english.stackexchange.com/questions/52750/is-it-correct-that-etc-can-not-be-used-together-in-a-sentence-with-for-examp)

File: website/content/en/blog/clusterlink-intro/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+## Introducing ClusterLink
+
+ClusterLink offers a secure and performant solution to interconnect services
+ across multiple clusters. It has a simple management model, built on the following
+ abstractions:
+
+- **Fabric**: a set of collaborating clusters, all sharing the same root of trust.
+ Clusters must be part of a fabric to enable multicluster networking.
+- **Peer**: a specific clusters in a fabric. Each peer is identified by a certificate,
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:59]:
```suggestion
- **Peer**: a specific cluster in a fabric. Each peer is identified by a certificate,
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+ decisions on service sharing and access control.
+- **Export**/**Import**: services must be explicitly shared by clusters before
+ they can be used. A service can be imported by any number of peers. To increase
+ availability or performance, a service can be exported by more than one peer.
+- **Access policies**: ClusterLink supports fine grained segmentation with a
+ "default deny" policy, adhering to "zero trust" principles. Access policies are
+ used to explicitly allow and deny communications. Affected workloads are defined
+ in terms of their attributes (such as location, environment, namespace or even
+ labels) and have two priorities, with privileged (i.e., administrator defined,
+ cluster scoped) policies evaluated before user defined namespaced policies.
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:70]:
```suggestion
 cluster scoped) policies evaluated before user-defined namespaced policies.
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+ standard Kubernetes services, ClusterLink integrates seamlessly with the Kubernetes
+ network model, and can work with any Kubernetes distribution, CNI and IP address
+ management scheme.
+
+The local service endpoints refer to **data plane** Pods, responsible for
+ workload-to-service secure tunnels to other clusters. The data plane uses
+ [HTTP CONNECT](https://en.wikipedia.org/wiki/HTTP_tunnel) with [mutual
+ TLS](https://en.wikipedia.org/wiki/Mutual_authentication#mTLS) for security.
+ The use of HTTPS over tcp/443 removes the need for VPNs and special firewall
+ configurations. Certificate based mTLS ensures guarantees in-transit data
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:92]:
```suggestion
 configurations. Certificate based mTLS guarantees in-transit data
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+ encryption and limits connection only other fabric peers. In addition,
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:93]:
```suggestion
 encryption and limits connection to only other fabric peers. In addition,
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+ microservices with other clusters and namespaces should be limited to those
+ belonging to the same unit, while also communicating with *exposed* services
+ belonging to other units.
+- services are owned by **different administrative domains** (e.g., different
+ development teams) and thus judicious sharing and more stringent access
+ controls across clusters are needed.
+- it is desirable to **increase scalability and limit information sharing** by
+ minimizing information exchanged between clusters. With ClusterLink, each
+ cluster manages its own naming and load balancing, requiring considerable
+ less cross cluster metadata for its communication.
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:116]:
```suggestion
 less cross-cluster metadata for its communication.
```

File: website/content/en/blog/clusterlink-intro/index.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+- there is a need for **separation of concerns** between network administrators
+ and application owners.
+
+## Getting started with ClusterLink
+
+To get started with ClusterLink, we invite you to explore the rest of the
+ documentation and familiarize yourself with its concepts and operation.
+ When ready, try out the [getting started tutorial]({{% ref "iperf" %}}).
+
+We would love to hear feedback and explore how we can make ClusterLink
```
Comment (zivnevo) [inline website/content/en/blog/clusterlink-intro/index.md:126]: Maybe explicitly mention somewhere that ClusterLink is an open-source project with a very permissive license.

**Review by zivnevo — COMMENTED** [review]
Very well written!
Says everything we want to say with not too many words.

Some nits below.

**Review by michalmalka — APPROVED** [review]

**Pull request merged**